packages/ButtonLink/ButtonLink.jsx

```diff
@@ -85,7 +55,11 @@ ButtonLink.propTypes = {
   /**
    * The style.
    */
-  variant: PropTypes.oneOf(['primary', 'secondary', 'inverted']),
+  variant: PropTypes.oneOf(['primary', 'secondary', 'inverted', 'standard', 'brand', 'danger']),
```

Comment:
`danger` should not be a valid variant for `ButtonLink` as it should only be used to perform an action and not as a navigation element.